### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by PopcornPaws**
## Description
- compilation flag (feature) added for the target Solana cluster in the wasm client
- `isIdUniqueWasm` added to help the frontend to check auction-id uniqueness
- docker scripts

### CODE REVIEW

File: client/src/get_auction.rs [hunk 1]
```diff
@@ -46,9 +46,10 @@ pub async fn get_auction(auction_id: String) -> Result<FrontendAuction, anyhow::
             }
         }
         TokenConfig::Token(ref data) => {
+            // TODO
             // get mint metadata
-            let mint_data = client.get_account_data(&data.mint).await?;
-            // get decimals
+            let mint_data = vec![0_u8; 200]; //client.get_account_data(&data.mint).await?;
```
Comment (gyozosz) [inline client/src/get_auction.rs:51]: This does not seem right. Did you mean to change it back once it worked?

File: gold-bot/src/main.rs [hunk 2]
```diff
@@ -120,8 +119,10 @@ async fn try_main(
         // read auction pool
         let (auction_pool_pubkey, _) =
             Pubkey::find_program_address(&auction_pool_seeds(), &GOLD_ID);
-        let auction_pool_data = client.get_account_data(&auction_pool_pubkey).await?;
-        let auction_pool: AuctionPool = try_from_slice_unchecked(&auction_pool_data)?;
+        let auction_pool = client
+            .get_and_deserialize_account_data::<AuctionPool>(&auction_pool_pubkey)
+            .await?;
+        //let auction_pool: AuctionPool = try_from_slice_unchecked(&auction_pool_data)?;
```
Comment (gyozosz) [inline gold-bot/src/main.rs:125]: Delete this comment

**Review by gyozosz — CHANGES_REQUESTED** [review]
See comments.

File: client/src/get_auction.rs (hunk 1 above)
Comment (PopcornPaws) [reply client/src/get_auction.rs:51]: Yep, nice catch

**Review by gyozosz — APPROVED** [review]

**Pull request merged**